### ISSUE TITLE
Infrastructure tasks

### ISSUE THREAD

**Issue by tacocatcodes** [safehavennetwork/safehaven_portal#67]
- [x] Create and configure new DO host for Secure Portal under SafeHaven DO account
- [x] Migrate Wordpress site to SafeHaven DO host
- [x] SSL cert for secure.thesafehavennetwork.org

Wordpress needs:
- nginx
- hhvm for PHP (yay fast PHP stuff!)

**Comment by themoffatt** [comment]
Added task for SSL cert

**Comment by tacocatcodes** [comment]
@themoffatt I will need the following info:
- 2 letter Country Code [US]
- State [IL]
- Locality [Chicago]
- Organization Name --- assuming Safe Haven?

I'm assuming the non-profit is registered in Chicago, but I need to know the legal name of the organization for the SSL cert.

**Comment by themoffatt** [comment]
I updated the tasks here a bit.  @laneatomic and I talked about it, and we're moving over to Digital Ocean for hosting.  AWS had been charging us over $40 a month for low traffic on a host with ~512Mb of memory.  Lane and I are more familiar and comfortable working with Digital Ocean.  If anyone has any questions on this move, feel free to ask.

We had 2 SSL certs waiting to be activated on our NameCheap account.   I setup the CSR and the and used DNS verification to complete the authentication.  We now have a cert to use!
